public/js/success.js

```diff
@@ -0,0 +1,52 @@
+function renderItem(cartElement, product, quantity) {
+  $(cartElement)
+    .find('.product-link')
+    .each(function () {
+      $(this).attr('href', `/product?id=${product.id}`)
+    })
+  $(cartElement).find('.image-placeholder').attr('src', product.image)
+  $(cartElement).find('.product-name').html(product.name)
+  $(cartElement).find('.quantity').text(quantity)
+  $(cartElement)
+    .find('.item-price')
+    .text(productService.renderPrice(product.price * quantity))
+}
+
+async function renderSummary() {
+  const template = await $.get('/components/order-summary-item.html')
+  products = await cartService.getCartProducts()
+
+  if (products.length < 1) {
+    console.log('umm')
+    location.replace('/404.html')
+  }
```

Comment:
Love the console.log, makes it look like I wrote it.

Reply:
oops. time to clean up the evidence. no witnesses